manifests/init.pp

```diff
@@ -80,11 +88,13 @@
             onlyif   => "! grep '^exclude=' /etc/yum.conf",
             user     => root,
             provider => shell,
-            require  => Exec['Ensure Service']
+            require  => Exec['Ensure Service'],
         }->
         exec { 'Yum Exclude gitlab-ci-multi-runner':
-            command  => "sed -i 's/^exclude=.*$/& gitlab-ci-multi-runner/' /etc/yum.conf",
-            onlyif   => "! grep '^exclude=.*gitlab-ci-multi-runner' /etc/yum.conf",
+            command  => "sed -i 's/^exclude=.*$/& gitlab-ci-multi-runner/' \
```

Comment:
I think 80 lines is an unreasonably short limit.  While I appreciate your work to bring us into the standard - I think this is more readable as a singe line.